day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,18 @@
+def fizzbuzz
+  (1..100).map do |n|
```

Comment:
Nice work here! Is there a reason why you used .map instead of .each?

Reply:
.each returns the original unchanged object (1..100) and, in the context of the way this code is written returns the entirety of the FizzBuzz strings as desired. .each could be utilized, but .map has the desired functionality and is cleaner.